src/components/Listing/Tracks/DelegationSuccessPopup.tsx

```diff
@@ -52,29 +62,43 @@ const DelegationSuccessPopup = ({ className, open, setOpen, tracks, address, isD
 		className={`${poppins.variable} ${poppins.className} ${isDelegate ? 'delegate' : 'undelegate'}`}
 		wrapClassName={className}
 		closeIcon={isDelegate ? <CloseIcon/> : <UndelegateCloseIcon/>}
-		onCancel={() => {setOpen(false); router.reload();}}
+		onCancel={() => {!isVote &&  router.reload() ; setOpen(false); }}
 		centered
 		footer={false}
 		maskClosable={false}
 	>
 		<div className='flex justify-center items-center flex-col -mt-[132px]'>
 			<SuccessIcon/>
-			<h2 className='text-[20px] font-semibold tracking-[0.0015em] mt-6'>{isDelegate ? 'Delegated successfully' : 'Undelegated successfully' }</h2>
+			<h2 className='text-[20px] font-semibold tracking-[0.0015em] mt-6'>{isDelegate ? `${title} successfully` : 'Undelegated successfully' }</h2>
 			{isDelegate && <div className='flex flex-col justify-center items-center gap-[18px]'>
 				{balance && <div className='text-pink_primary text-[24px] font-semibold'>{formatBalance(balance.toString(),{ forceUnit: unit })}</div>}
+				{
+					vote === EVoteDecisionType.SPLIT && <div className=' flex flex-wrap justify-center font-normal text-sm text-[#243A57]'> <span className='mr-3'> Aye: {ayeVoteValue ? formatBnBalance(ayeVoteValue,{ withUnit:true },network): 0}</span> <span>Nay: {nayVoteValue ? formatBnBalance(nayVoteValue,{ withUnit:true },network) : 0}</span></div>
+				}
+				{
+					vote === EVoteDecisionType.ABSTAIN &&  <div className='flex flex-wrap justify-center font-normal text-sm text-[#243A57]'> <span className='mr-3'> Abstain: {abstainVoteValue ? formatBnBalance(abstainVoteValue,{ withUnit:true },network): 0}</span>  <span className='mr-3'> Aye: {ayeVoteValue ? formatBnBalance(ayeVoteValue,{ withUnit:true },network): 0}</span> <span>Nay: {nayVoteValue ? formatBnBalance(nayVoteValue,{ withUnit:true },network) : 0}</span></div>
```

Comment:
use formatedBalance instead of take ref>>> src/components/DelegationDashboard/DelegateCard.tsx

Reply:
resolved